src/components/ContributorCard.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import Img from "gatsby-image"
+
+const ContributorCard = ({ name, contributions, avatar, website }) => (
+    <a href={website} className="flex p-3 my-2 rounded bg-white w-15">
```

Comment:
I wonder if we want to show this list in two places: here and in its own page.

In this use case, I would vote for a more compact grid that does not link to websites. On its own page, I think its great to share more details